daemon/logger/awslogs/cloudwatchlogs_test.go

```diff
@@ -938,6 +938,61 @@ func TestCollectBatchClose(t *testing.T) {
 	}
 }
 
+func TestEffectiveLen(t *testing.T) {
+	tests := []struct {
+		str            string
+		effectiveBytes int
+	}{
+		{"Hello", 5},
```

Comment:
Might make sense to add empty string as a test case also.

Reply:
Done.